talent/onboarding/README.md

```diff
@@ -89,4 +89,4 @@ At the San Francisco office, we provide:
 
 #### San Francisco Conference Rooms
 
-- Since we're sharing the space with Holberton, the rooms are on use them as you need basis (they can't be reserved).
\ No newline at end of file
+- Since we're sharing the space with Holberton, the rooms are on use them as you need basis (they can't be reserved).
```

Comment:
@estherrgarcia  The part of "the rooms are on use them as you need basis" is not very clear to me

Reply:
@tsolakoua it means, that we can make use of their conference rooms but we don't have a dedicated source{d} conf. room there yet.

Reply:
@eiso thanks! it seems that there is something wrong with the syntax of the sentence and doesn't make it very clear, at least to me. Not sure if needs to be changed.

Reply:
Please feel free to change it ;)